optimizer_wrapper.rb

```diff
@@ -585,6 +585,20 @@ def self.build_csv(solutions)
   private
 
   def self.check_result_consistency(expected_value, results)
+    [results].flatten(1).each{ |result|
+      waiting_time_map = result[:routes].map{ |route| route[:total_waiting_time] }.compact
+      duration_map = result[:routes].map{ |route|
+        route[:activities].map{ |act| act[:departure_time] && (act[:departure_time] - act[:begin_time]) }.compact
+      }
+
+      next if result[:total_time] == result[:total_travel_time] + waiting_time_map.sum + duration_map.flatten.sum
```

Comment:
Can you add another "non-production" before this `next` so that we don't have any negative weights in any routes?

Something similar to
```ruby
if result[:routes][:activities].any?{ |a| a[waiting_time].to_i < 0 }
  log 'Computed waiting times are invalid', level: :warn 
  # I wish the level warn would raise automatically on non-production environments 
  # so that we wouldn't need the next line
  raise 'Computed waiting times are invalid' if ENV['APP_ENV'] != 'production'
end
```